gcsio/pom.xml

```diff
@@ -163,6 +163,11 @@
       <artifactId>gson</artifactId>
       <scope>compile</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.hadoop</groupId>
```

Comment:
GCSIO library can not depend on anything in `org.apache.hadoop` - this is standalone GCS client intended to be usable outside of Hadoop